src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -15,6 +15,7 @@ public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
     private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
```

Comment:
still need this?

Reply:
affirm sir, existing AB3 App still uses this UserPrefs class so we will retain until the AB3 purge